tests/integration/assignments/test_polls.py

```diff
@@ -103,8 +103,9 @@ def test_simple(self):
                 "pollmethod": "YNA",
                 "type": "named",
                 "assignment_id": self.assignment.id,
+                "onehundred_percent_base": "YN",
```

Comment:
I would rather use the constants instead of the string, since that's what we have them for (same for all similar lines below)